microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/install.md

```diff
@@ -0,0 +1,295 @@
+
+Below are specific steps to install a couple different web servers.
+
+
+<!-- ------------------------------ -->
+#### Set up a localhost server by using npx http-server
+
+If you installed Node.js and npm, as described above, `npx http-server` is a simple way to start a local web server.  For information, see [http-server: a simple static HTTP server](https://www.npmjs.com/package/http-server).
+
+
+1. In Visual Studio Code, select **File** > **Open Folder** > open the directory that contains the `.html`, `.css`, and `.js` files for a webpage, such as `C:\Users\myusername\Documents\GitHub\Demos\demo-to-do\`.
+
+   In the above example path:
+   *  `\Documents\GitHub\` is where the `Demos` repo was cloned to.
+   *  `\Demos\` is the GitHub repo that's used for examples in the Microsoft Edge Developer documentation.
+   *  `\demo-to-do\` is one of the sample directories in the repo.
+
+1. In Visual Studio Code, select **View** > **Terminal**.  Or, to keep the web server running regardless of Visual Studio's state, open a command prompt outside of Visual Studio Code, such as `git bash`.
+
+1. `cd` into the the folder you want to serve via http.
+
+   For example, on Windows:
+
+   ```
+   cd C:\Users\myusername\Documents\GitHub\Demos\demo-to-do\
+   ```
+
+<!-- if you cd to 
+C:\Users\myusername\Documents\GitHub\Demos\
+you would then specify
+http://localhost:8080/demo-to-do/
+but it's fine usually to serve out a single sample dir
+but serving out the entire \Demos\ dir gives a useful dir listing at:
+   http://10.0.1.8:8080
+   http://127.0.0.1:8080
+-->
+
+   **Start the server (npx http-server):**<!-- make prominent/findable -->
+
+1. Enter the command `npx http-server`:
+
+   ```
+   npx http-server
+   ```
+
+   A local webserver starts on port 8080.
+
+   You might receive a message like:
+
+   ```
+   Need to install the following packages:
+     http-server@14.1.1
+   Ok to proceed? (y)
+   ```
+
+1. Enter **y**.
+
+   Windows prompts whether to allow Node to communicate on the network:
+
+   ![Allowing Node to communicate on the network](./install-images/allow-node-communicate-network.png)
+
+1. Select the **Private networks** checkbox, and then click the **Allow access** button.
+
+   Information about the server and localhost URL is displayed, such as:
+
+   ```
+   Starting up http-server, serving ./
+   
+   Available on:
+   http://10.0.1.8:8080
+   http://127.0.0.1:8080
+   Hit CTRL-C to stop the server
+   ```
+   
+   The displayed URLs are equivalent to [http://localhost:8080/](http://localhost:8080/).
+
+Next, see [Get started using the DevTools extension for Visual Studio Code](./get-started.md) for a step-by-step walkthrough.
+
+
+<!-- ------------------------------ -->
+#### Set up a localhost server by using IIS on Windows
```

Comment:
I'm really not sure about giving this option too. It feels like telling users how to start a web server with http-server is enough. Giving multiple options feels a bit too much to me. Our docs are about the VSCode extension, not about how to run web servers locally.

Reply:
removed